Gulpfile.js

```diff
@@ -102,6 +113,8 @@ function generateTypeHelpers(helperKind) {
       })
     )
     .pipe(gulp.dest(dest));
+
+  return finish(stream);
```

Comment:
@JLHwung This is a bug introduced by the recent gulpfile changes: we were returning a stream instead of a promise, and we were passing those streams to `Promise.all`.

Reply:
Ah good catch!